.github/workflows/push-fe-build-to-main.yml

```diff
@@ -0,0 +1,53 @@
+name: Push build to Main
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "frontend/**"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Remove old build files
+        run: |
+          find backend/src/zango/assets/app_panel/js -name "build.*.min.js" -type f -delete
+          find backend/src/zango/assets/app_panel/js -name "build.*.min.js.LICENSE.txt" -type f -delete
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: "16"
+
+      - name: Install dependencies
+        run: |
+          cd frontend
+          yarn install
+
+      - name: Building app
+        run: |
+          cd frontend
+          CI=false yarn build
+
+      - name: Setup Git for commit
+        run: |
+          git config user.name "github-actions"
+          git config user.email "actions@zango.com"
```

Comment:
can we use `${{ secrets.USER_EMAIL }}` here? same as we have used in https://github.com/Healthlane-Technologies/Zango/blob/fe-build-test/.github/workflows/sync-docs-to-docs-hub-main.yml#L42C23-L42C48